src/context/Theme.tsx

```diff
@@ -197,6 +205,21 @@ export const jsonFormsGroupHeaders = {
     dark: 'transparent',
 };
 
+export const reflexSplitterBackground = {
+    light: slate[25],
+    dark: slate[200],
+};
+
+export const alternativeReflexContainerBackground = {
```

Comment:
Currently, the _alernative_ styles are only applied to the `ListAndDetails` component.